bindings/python/CHANGELOG.rst

```diff
@@ -4,7 +4,12 @@ Changelog
 Changes in Version 1.2.0
 ------------------------
 
+- Add support for the "kmip" KMS provider.
 - Add MongoCryptKmsContext.kms_provider property.
+- Bundle libmongocrypt 1.3.0 in release wheels.
+- **Remove support for libmongocrypt 1.2, libmongocrypt >=1.2 is now
```

Comment:
Do you mean remove support for libmongocrypt <1.2, or that libmongocrypt > 1.2 is required?

Reply:
Fixed.